Разметка главной страницы

NatashaStr opened the pull request:
---
:mortar_board: [Разметка главной страницы](https://up.htmlacademy.ru/htmlcss/33/user/1934365/tasks/3)

:boom: https://htmlacademy-htmlcss.github.io/1934365-sedona-33/2/

---
:mortar_board: [Разметка внутренней страницы](https://up.htmlacademy.ru/htmlcss/33/user/1934365/tasks/4)

NatashaStr committed:
Добавляет header

NatashaStr committed:
Добавляет main и footer

NatashaStr committed:
Убирает лишние пробелы

NatashaStr committed:
Добавляет страницу каталога

Ustincheg commented on index.html:
```diff
@@ -2,11 +2,121 @@
-
+<header class="page-header">
+  <nav class="navigation">
+    <p class="visually-hidden"><a class="navigation-link" href="#">Логотип</a></p>
```
Тэг p в данном случае лишний, следует оставить только тэг а

NatashaStr replied:
исправлено, добавила span c классом visually-hidden

Ustincheg commented on catalog.html:
```diff
@@ -0,0 +1,216 @@
+  <body>
+    <header class="page-header">
+        <nav class="navigation">
+          <p class="visually-hidden"><a class="navigation-link" href="index.html">Логотип</a></p>
```
Тэг p в данном случае лишний, следует оставить только тэг а

Ustincheg commented on index.html:
```diff
@@ -2,11 +2,121 @@
+    </ul>
+  </nav>
+</header>
+    <main class="main-index">
```
первым тегом внутри main следует создать тег h1;
Внутрь h1 следует разместить текст передающий смысл всей страницы, что-то вроде "Сервис подбора гостиниц в городе Седона";
Такой заголовок будет скрытым "visually-hidden"

NatashaStr replied:
исправлено. Сначала сделала правильно, потом решила посмотреть код на сайте и переделала) там вообще весь сайт на дивах сделан, так что больше туда не заглядываю)

Ustincheg commented on index.html:
```diff
@@ -2,11 +2,121 @@
+      <img class="greeting-image" src="" alt="Welcome">
+      <img class="greeting-image" src="" alt="to the gorgeous">
+      <img class="logo" src="" alt="Логотип города Седона">
+<h1 class="greeting-title">Седона — небольшой городок в Аризоне,<br>заслуживающий большего!</h1>
```
Согласен, внешне этот заголовок похож на главный заголовок страницы, но по смыслу это просто текст.
Здесь следует использовать тэг p
И опустить этот текст в секцию ниже "reasons"

NatashaStr replied:
Исправлено, добавила тег h2 в эту секцию с заголовком Причины выбрать Седону. Теги, которые до этого были h2 поменяла на h3

Ustincheg commented on index.html:
```diff
@@ -2,11 +2,121 @@
+  </nav>
+</header>
+    <main class="main-index">
+      <section class="greeting">
```
У каждой секции следует создать заголовок второго уровня, который отображает смысл секции, если такого нет в макете, то их следует сделать скрытыми. В данном случае это будет h2 "Приветствие" h2  или h2 "Главный баннер" h2

NatashaStr replied:
Исправлено

Ustincheg commented on index.html:
```diff
@@ -2,11 +2,121 @@
+      <img class="logo" src="" alt="Логотип города Седона">
+<h1 class="greeting-title">Седона — небольшой городок в Аризоне,<br>заслуживающий большего!</h1>
+  </section>
+  <section class="reasons">
```
У каждой секции следует создать заголовок второго уровня, который отображает смысл секции, если такого нет в макете, то их следует сделать скрытыми. В данном случае это будет h2 "преимущества посещения Седоны" h2

Ustincheg commented on index.html:
```diff
@@ -2,11 +2,121 @@
+<p class="reasons-description">Рассмотрим 5 причин, по которым Седона круче, чем Гранд-Каньон!</p>
+<ul class="reasons-list">
+  <li class="reasons-item">
+    <h2 class="reasons-title">Настоящий городок</h2>
```
здесь и далее внутри текущей секции меняем h2 на h3

Ustincheg commented on index.html:
```diff
@@ -2,11 +2,121 @@
+</ul>
+</section>
+<section class="search">
+  <h2 class="search-title">Заинтересовались?</h2>
```
заголовок должен отражать смысл секции, так что нужно сделать отдельный заголовок "ПОИСК ГОСТИНИЦЫ В СЕДОНЕ". текст "заинтересовались" - завернуть в тег p

NatashaStr replied:
Исправлено. есть вопрос для именования классов к тегам р, если 2 тега р идут подряд. Можно ли написать так:
p class="**search-description-first"**Заинтересовались?/p
  p class=**"search-description-second"**Укажите предполагаемые даты поездки, и мы покажем вам лучшие предложения гостиниц в Седоне/p
Либо нужно какой-то доп класс к ним придумать?

Ustincheg commented on index.html:
```diff
@@ -2,11 +2,121 @@
+    </section>
+    <section class="footer-logo">
+      <h2 class="footer-logo-title">Логотип</h2>
+          <p class="footer-logo">.html academy</p>
```
логотип будет картинкой img

Ustincheg commented on catalog.html:
```diff
@@ -0,0 +1,216 @@
+      </header>
+      <main class="main-inner">
+          <header class=inner-header>
+              <h1 class="inner-header-title">Гостиницы Седоны</h1>
```
на этой странице h1 в макете более соответствует смыслу страницы, можно оставить

NatashaStr replied:
Т.е. ничего не меняем, правильно?

Ustincheg commented on catalog.html:
```diff
@@ -0,0 +1,216 @@
+                      <a class="breadcrumbs-link">Гостиницы</a>
+                  </li>
+              </ul>
+          </header>
```
внутри main еще один header? Лучше исправить на div и положить внутрь секции filters - у них один бэкграунд

NatashaStr replied:
исправлено

Ustincheg commented on catalog.html:
```diff
@@ -0,0 +1,216 @@
+                <span class="visually-hidden">Показать список</span>
+            </a>
+          </section>
+          <section class="hotels">
```
Следует добавить скрытый заголовок

NatashaStr replied:
Исправлено

Ustincheg reviewed (commented):
Наталья, будут вопросы по замечаниям - пишите в телеграмм 89023160881

NatashaStr committed:
корректирует код в соответствии с комментариями

1. В файле index.html исправлен код в соответствии с комментариями.
2. В файле catalog.html ещё добавлен звёздный рейтинг к карточкам гостиниц
